lab-matt/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This looks great, the only thing I'd add since you're using Mocha for your tests would be to specify that "scripts" > "test": "mocha" rather than "echo \"Error: no test specified\" && exit 1" since you do have tests and they are run with mocha.  That way from your assignment root directory a user could run your tests with `npm test`.  Mocha is a pretty simple thing to run on its own, but when you have more complex testing procedures this can help simplify the task for users.